apps/jobs/views.py

```diff
@@ -360,7 +360,14 @@ def leaderboard(request, challenge_phase_split_id):
 
     leaderboard_labels = challenge_phase_split.leaderboard.schema['labels']
     for item in distinct_sorted_leaderboard_data:
-        item['result'] = [item['result'][index] for index in leaderboard_labels]
+        result = []
+        for index in leaderboard_labels:
+            try:
+                result.append(item['result'][index])
+            except:
+                response_data = {'error': 'Sorry, label {} not found in leaderboard schema!'.format(index)}
```

Comment:
Please change response_data to `{'error': 'Sorry, label {} is not found in leaderboard schema'.format(index)}`